legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/BaseGitLabApi.java

```diff
@@ -585,63 +537,50 @@ protected LegendSDLCServerException buildException(Exception e, Supplier<String>
 
     protected LegendSDLCServerException buildException(Exception e, Supplier<String> forbiddenMessage, Supplier<String> notFoundMessage, Supplier<String> defaultMessage)
     {
-        return buildException(e,
-                (forbiddenMessage == null) ?
-                        null :
-                        ex -> Optional.ofNullable(forbiddenMessage.get()).map(m -> StringTools.appendThrowableMessageIfPresent(m, ex)).orElse(null),
-                (notFoundMessage == null) ?
-                        null :
-                        ex -> notFoundMessage.get(),
-                (defaultMessage == null) ?
-                        null :
-                        ex -> Optional.ofNullable(defaultMessage.get()).map(m -> StringTools.appendThrowableMessageIfPresent(m, ex)).orElse(null));
+        return buildException(e, (forbiddenMessage == null) ? null : ex -> Optional.ofNullable(forbiddenMessage.get()).map(m -> StringTools.appendThrowableMessageIfPresent(m, ex)).orElse(null), (notFoundMessage == null) ? null : ex -> notFoundMessage.get(), (defaultMessage == null) ? null : ex -> Optional.ofNullable(defaultMessage.get()).map(m -> StringTools.appendThrowableMessageIfPresent(m, ex)).orElse(null));
     }
 
     protected LegendSDLCServerException buildException(Exception e, Function<? super GitLabApiException, String> forbiddenMessage, Function<? super GitLabApiException, String> notFoundMessage, Function<? super Exception, String> defaultMessage)
     {
-        return processException(e,
-                Function.identity(),
-                glae ->
+        return processException(e, Function.identity(), glae ->
```

Comment:
I think the code is more readable in this case when there's one argument per line.